source/auth.txt

```diff
@@ -240,12 +240,36 @@ To learn more, see the blog :website:`Manage
 MongoDB Atlas Database Secrets in HashiCorp Vault 
 </blog/post/manage-atlas-database-secrets-hashicorp-vault>`.
 
+.. _arch-center-admin-api-recs:
 
 Recommendations for {+atlas-admin-api+} 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|service| provides |api| key-based authentication to securely manage
-programmatic access. It uses |http| Digest authentication to protect requests.
+|service| provides two ways to authenticate to the {+atlas-admin-api+}:
+
+- :atlas:`Service accounts </api/service-accounts-overview/>`
+  (currently available as a `Preview feature 
+  <https://www.mongodb.com/resources/products/early-access>`__)
```

Comment:
The parentheses is rendering as part of the link, maybe we can use the :website: directive to fix this:

```suggestion
  (currently available as a :website:`Preview feature 
  </resources/products/early-access>`)
```